### ISSUE TITLE
MSFT-264Keyboard access tab operational missing

### ISSUE THREAD

**Issue by veshu** [veshu/testjiraimport#5]
**Steps to Reproduce**
1) Go to the detail page 'https://www.topcoder.com/challenges/30095347?tab=details'
2) Find 'ABOUT US' in footer of the page (use virtual key to check)
3) Use tab to navigate from virtual keyboard
4) Observe the tab between instagram icon and support button are missing 
5) tab after instagram icon hit enter no action perform then next tab support button popup.

**Expected Result**
Every navigate tab from keyboard must be perform some action

**Actual Result**
Missing tab action while navigate via keyboard

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome
**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=8ab28edc%2Dd11f%2D4bb3%2Db9cf%2Da743b86a634f&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fkmurti%2FAccessibility%2F9
